src/app/component/core/header-new/header-new.component.html

```diff
@@ -56,29 +59,118 @@
                             {{ 'user.lower-nav-bar.about-us' | translate }}
                           </a>
                         </li>
+                        <div *ngIf="!isLoggedIn; then userLoggedInSmall else userNotLoggerInSmall"></div>
+                        <ng-template #userLoggedInSmall>
+                        <li class="mobile-vis">
+                            <a class="sign-in-link"
+                               (click)="openSingInWindow()">
+                              {{ "top-bar.sign-in" | translate }}
+                            </a>
+                        </li>
+                        </ng-template>
+                        <ng-template #userNotLoggerInSmall>
+                          <li class="mobile-vis tertiary-global-button">Nadiia Steblivets</li>
+                          <li class="mobile-vis">
+                            <a (click)="openDialog()">
+                            {{ "nav-bar.favorite-places" | translate }}
+                             </a>
+                          </li>
+                          <li class="mobile-vis">
+                            <a (click)="openSettingDialog()">
+                            {{ "nav-bar.user-settings" | translate }}
+                             </a>
+                          </li>
+                          <li class="mobile-vis">
+                              <a (click)="signOut()">
+                            {{ "nav-bar.sign-out" | translate }}
+                             </a>
+                          </li>
+                        </ng-template>
+                        <li class="mobile-vis"
+                            *ngIf="!isLoggedIn">
+                          <div class="create-button secondary-global-button">
+                            <span>Sign up</span>
+                          </div>
+                        </li>
                       </ul>
-                    </nav>
                   </div>
         <div class="navigation-menu-right">
               <ul>
-                          <li class="search"><a  href="#">    <img src="assets/img/icon/search.png" alt="search">  </a></li>
-              <li>    <div id="switcher-wrapper">
-                <img id="arrow" [src]="arrow" alt="v">
-                <select [(ngModel)]="language" (change)="changeCurrentLanguage()" id="language-switcher">
-                  <option class="language-option" [ngValue]="'uk'">Ua</option>
-                  <option class="language-option" [ngValue]="'ru'">Ru</option>
-                  <option class="language-option" [ngValue]="'en'">En</option>
+               <li class="search"
+                   [class.destroy]="onToggleBurgerMenu">
+               <a  href="#">
+                 <img src="assets/img/icon/search.png" alt="search">
```

Comment:
Please write attributes in separate rows

Reply:
+